wrangler-core/src/main/java/co/cask/wrangler/clients/SchemaRegistryClient.java

```diff
@@ -98,16 +99,18 @@ public static SchemaRegistryClient getInstance(ExecutorContext context) {
   /**
    * Retrieves schema provided a schema id and version of the schema.
    *
-   * @param id of the schema.
-   * @param version of the schema.
+   * @param id the schema id
+   * @param version the schema version
    * @return {@link Response} of the schema.
    * @throws URISyntaxException thrown if there are issue with construction of url.
    * @throws IOException throw when there are issues connecting to the service.
    * @throws RestClientException thrown when there are issues with request or response returned.
    */
-  public byte[] getSchema(String id, long version)
+  public byte[] getSchema(NamespacedId id, long version)
```

Comment:
should we change javadoc to reflect this change?

Reply:
javadoc doesn't change because it's still the id of the schema.